.github/workflows/yamllint/.yamllint.yml

```diff
@@ -3,6 +3,7 @@ extends: default
 ignore: |
   /themes/classic/_dev/node_modules/
   /themes/node_modules/
+  .github/ISSUE_TEMPLATE/
```

Comment:
Why not fixing them ?

Reply:
Because they are not broken

Reply:
I guess we could use something like that:
```YAML
value: >
        Thank you for taking the time to fill out this bug report.
        Please take the time to provide as much details as possible.
        Following our [bug report guidelines](https://devdocs.prestashop.com/1.7/contribute/contribute-reporting-issues/#best-practices-for-writing-an-issue)
        will help maintainers qualify your report faster.
```
But ignoring this file is okay for me too, I'll approve ;)

Reply:
> I guess we could use something like that:

I tried but I think the link in https://github.com/PrestaShop/PrestaShop/edit/develop/.github/ISSUE_TEMPLATE/2_feature_request.yml is still too long and cannot be split

```
[searched in existing features request](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3AImprovement%2CFeature%2CDeveloper+Feature+)
```